nusantara/utils/common_parser.py

```diff
@@ -15,3 +19,102 @@ def load_conll_data(file_path):
             sentence = []
             seq_label = []
     return dataset
+
+
+def load_ud_data(filepath):
+    """
+    Load and parse conllu data.
+
+    Proposed by @fhudi for issue #34 and #9.
+
+    :param filepath: file path
+    :return: generator with schema following CONLLU
+    """
+    dataset_raw = parse(open(filepath).read())
+    return map(lambda sent: {**sent.metadata, **pd.DataFrame(sent).to_dict(orient="list")}, dataset_raw)
+
+
+def load_ud_data_as_nusantara_kb(filepath):
+    """
+    Load and parse conllu data, followed by mapping its elements to Nusantara Knowledge Base schema.
+
+    Proposed by @fhudi for issue #34 and #9.
+
+    :param filepath: file path
+    :return: generator for Nusantara KB schema
+    """
+    dataset_source = list(load_ud_data(filepath))
+
+    def as_nusa_kb(tokens):
+        sent_id = tokens["sent_id"]
+        offsets = get_span_offsets(tokens["form"], tokens["text"])
+        return {
+            "id": sent_id,
+            "entities": [
+                {
+                    "id": f"{sent_id}_EntID_{ent_id}",
+                    "type": ent_type,
+                    "text": [ent_text],
+                    "offsets": [offset],
+                    "normalized": [
+                        {
+                            "db_name": norm_text,
+                            "db_id": None,
+                        }
+                    ],
+                }
+                for (ent_id, ent_type, ent_text, offset, norm_text) in zip(tokens["id"], tokens["upos"], tokens["form"], offsets, tokens["lemma"])
+            ],
+            "relations": [
+                {
+                    "id": f"{sent_id}_RelID_{rel_id}",
+                    "type": rel_type,
+                    "arg1_id": f"{sent_id}_EntID_{rel_child}",
+                    "arg2_id": f"{sent_id}_EntID_{rel_parent}",
+                    "normalized": [],
+                }
+                for rel_id, (rel_type, rel_child, rel_parent) in enumerate(zip(tokens["deprel"], tokens["id"], tokens["head"]))
+            ],
+            "events": [],
+            "passages": [],
```

Comment:
This one needs to be filld with the original text, for example for PUD data (https://raw.githubusercontent.com/UniversalDependencies/UD_Indonesian-PUD/master/id_pud-ud-test.conllu), the passsages for the first example should be:
```
“Meski kebanyakan transisi digital yang terjadi di Amerika Serikat belum pernah terjadi sebelumnya, transisi kekuasaan yang damai tidaklah begitu,” tulis asisten khusus Obama, Kori Schulman di sebuah postingan blog pada hari Senin.
```

Which comes from the second line on the file:
```
# text = “Meski kebanyakan transisi digital yang terjadi di Amerika Serikat belum pernah terjadi sebelumnya, transisi kekuasaan yang damai tidaklah begitu,” tulis asisten khusus Obama, Kori Schulman di sebuah postingan blog pada hari Senin.
```